packages/lodestar/src/chain/blocks/verifyBlock.ts

```diff
@@ -199,11 +209,59 @@ export async function verifyBlockStateTransition(
         //    eventually win in fork-choice as other validators vote on VALID blocks.
         // Once EL catches up again and respond VALID, the fork choice will be updated which
         // will either validate or prune invalid blocks
+        //
+        // When to import such blocks:
+        // From: https://github.com/ethereum/consensus-specs/pull/2770/files
+        // A block MUST NOT be optimistically imported, unless either of the following
+        // conditions are met:
+        //
+        // 1. The justified checkpoint has execution enabled
+        // 2. The current slot (as per the system clock) is at least
+        //    SAFE_SLOTS_TO_IMPORT_OPTIMISTICALLY ahead of the slot of the block being
+        //    imported.
+        const justifiedBlock = chain.forkChoice.getJustifiedBlock();
+        const clockSlot = getCurrentSlot(chain.config, postState.genesisTime);
+        if (
+          justifiedBlock.executionStatus === ExecutionStatus.PreMerge ||
```

Comment:
maybe `&&`? what I read is if one of these conditions is not met, it's ok for an optimistic import

Reply:
condition for safe import is justified block is not execution enabled and block slot is SAFE_SLOTS_TO_IMPORT_OPTIMISTICALLY behind clock slot, so this condition is !(Not) of that,
i.e. throw error if justified block is pre merge or blockslot is within SAFE_SLOTS_TO_IMPORT_OPTIMISTICALLY from clockSlot, then one can't import the block (throw error)

Reply:
ok I got what you mean, it won't let a syncing node which is PreMerge to import blocks even though merge has happened and is way ahead :+1: